client/lib/setup/authorization.js

```diff
@@ -0,0 +1,16 @@
+
+export const requireLogin = (req, res, next) => {
+  if (req.user) {
+    next()
+  } else {
+    res.redirect('/login')
+  }
+}
+
+export const requireChannel = (req, res, next) => {
+  if (req.user && req.user.get('current_channel')) {
```

Comment:
Because we require a user _and_ a channel here, the above middleware is actually redundant.

Reply:
Yeah, that's a good point!

Reply:
Mm there's a subtle difference - no user means we should redirect to the login screen, whereas no channel means we should log the user out. Logging the user out might be a little extreme for having no channel, but there is a distinction. The logout helps in case the user's session is stale.

Reply:
Ah, good call! Thanks for pointing that out.